### ISSUE TITLE
Related Resources Text Features an Option that Does not Exist

### ISSUE THREAD

**Issue by ClaraCogswell** [hydroshare/hydroshare#4861]
**Description of the bug**
Above related resources on the resource landing page is the text "If this resource is related to other documents, datasets, websites, etc., you can record those relationships here as related resources. For example, if this resource is cited by a paper you published, select a relationship type of "The content of this resource serves as the data for" and then provide a full text citation for the paper that used the data, including the paper's DOI encoded as an HTTP URL if available. This records that this resource is the data for the paper that cited it. If you enter a URL as part of the citation, it will be converted to an active link. If the data in your resource was derived from another source or dataset, you can cite or acknowledge those relationships here." 

Problem is this line: "etc., you can record those relationships here as related resources. For example, if this resource is cited by a paper you published, select a relationship type of "The content of this resource serves as the data for"" Since the option "The content of this resource serves as the data for" does not exist. 

Steps to reproduce the bug:
1. Go to any resource, and view the information text above related resources. 

**Expected behavior**
Text should be an accurate reflection of functionality. 

**Additional information**
We should either add back this option or change the text to an existing option.

**Comment by horsburgh** [comment]
Not sure how that relationship type got removed. It should not have been. The list of relationship types we wanted are in this Google Doc:

https://docs.google.com/spreadsheets/d/1zcJcsxeTxik5dyuCJAkOooSfqbk1kVmjolZo6NYwIpA/edit#gid=0 

isDataFor is clearly one of the ones we wanted to keep using.

**Comment by devincowan** [comment]
thanks @horsburgh good to know, because the code makes this look pretty darn intentional! :(
I do see in that doc you sent that there is a note:

> Could also replace this with Dublin Core "isReferencedBy." This term is more specific and I've used it, but it could be replaced.
(column F)

So, as part of [1.52 release](https://github.com/hydroshare/hydroshare/issues/4458), we ran a migration on our data `migrate_relation_types` which moved all of the 'isDataFor' relations to 'isReferencedBy'. Looks like that release was ~2/1/2022 so I doubt that it is worth the time to go back and repair the relations that were impacted...

**Comment by horsburgh** [comment]
@devincowan - I agree with your assessment. This is one of those choices that was made in the implementation. If we stick with the relationship types we have, then the descriptive text needs to change as Clara suggested in this issue. I suggest changing to:

> If this resource is related to other documents, datasets, websites, etc., you can record those relationships here as related resources. For example, if this resource is cited by a paper you published, select a relationship type of "The content of this resource is referenced by" and then provide a full text citation for the paper that used the data, including the paper's DOI encoded as an HTTP URL if available. If you enter a URL as part of the citation, it will be converted to an active link. If the data in your resource was derived from another source or dataset, you can cite or acknowledge those relationships here."

**Comment by devincowan** [comment]
looks good to me, I will

- change the text on the Landing Page as suggested
- ensure 'isDataFor' is eradicated from the codebase
- check to ensure no relations currently exist on prod with the 'isDataFor' type

**Comment by devincowan** [comment]
Looks like there is only one 'isDataFor' relation in our system:

> rel = Relation.objects.filter(type="isDataFor").all()
> rel.count()
> 1
> rel.first().dict
> {'Relation': {'id': 3, 'object_id': 119, 'content_type': 78, 'type': 'isDataFor', 'value': 'http://www.hydroshare.org/resource/877bf9ed9e66468cadddb229838a9ced/'}}
@dtarb owns this resource.
I propose we migrate this relation to 'isReferencedBy'

**Comment by dtarb** [comment]
@devincowan 
I checked https://www.hydroshare.org/resource/b7822782896143ca8712395f6814c44b/.  This has reference isDerivedFrom that is correct and still valid.
I checked https://www.hydroshare.org/resource/877bf9ed9e66468cadddb229838a9ced/  which is the resource referenced by the above isDerivedFrom.  This had in the bag isDataFor.  I was able to edit this reference and now it is "isReferencedBy", so if this is the only instance then migrating may be unnecessary.
